src/qt/bitcoingui.h

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2011-2015 The Bitcoin Core developers
+// Copyright (c) 2011-2019 The Bitcoin Core developers
```

Comment:
Be consistent! (ie do all copyright updates or none of them)

Reply:
I updated all that changed and didn't have a "Dash Core" copyright. The ones with "Dash Core" are already saying 2021, so they are good. Is it better to drop all of them?

Reply:
this is a nit (ie not blocking and doesn't NEED to be done for this PR to be approved), but I would prefer either updating all of the bitcoin copyrights or none of them in this PR

Reply:
I'll drop these changes. No worries.